shell/assets/translations/en-us.yaml

```diff
@@ -5795,6 +5795,37 @@ user:
         placeholder: e.g. This account is for John Smith
   list:
     errorRefreshingGroupMemberships: Error refreshing group memberships
+  retention:
+    button:
+      label: User retention settings
+    edit:
+      title:
+        header: "Users: Settings"
+        pre: "Users:"
+        post: Settings
+        subTitle: User retention
+      form:
+        disableAfter:
+          checkbox: Disable user accounts after an inactivity period (duration since last login)
+          input:
+            label: Inactivity period
+            tooltip: Uses duration units (i.e. use 30d for 30 days)
+        deleteAfter:
+          checkbox: Delete user accounts after an inactivity period (duration since last login)
+          input:
+            label: Inactivity period
+            tooltip: Uses duration units (i.e. use 30d for 30 days)
```

Comment:
Probably the nitpickiest comment I've ever left on a PR, but technically this should be `e.g.` ('for example') not `i.e.` ('in other words')

Reply:
I think you're right. 

My intent for going with `i.e.` was to clarify what was just said. This should be a valid use case, but I think the clarification text doesn't fit because it's not a concrete definition of what was previously said. `e.g.` definitely fits better because "use 30d for 30 days" is definitely an example.